### PR TITLE
PLF-7379 Duplicated rows in jbid_io_rel table

### PR DESCRIPTION
**Pull request by sayachi**

### CODE REVIEW

**Comment by sayachi** [comment]
@boubaker This problem is reproduced only in PostgreSQL and HSQL databases. Should i add a unit test for HSQL ?

File: component/identity/src/test/java/org/exoplatform/services/organization/TestOrganization.java [hunk 1]
```diff
@@ -60,6 +64,10 @@
     protected static final String DEFAULT_PASSWORD = "defaultpassword";
     protected static final String DESCRIPTION = " Description";
 
+    private PortalContainer manager;
+    private Connection connection;
+    private Statement st;
```
Comment (aboughzela) [inline component/identity/src/test/java/org/exoplatform/services/organization/TestOrganization.java]: Please remove unused attribues

File: component/identity/src/test/java/org/exoplatform/services/organization/TestOrganization.java [hunk 2]
```diff
@@ -21,12 +21,15 @@
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
+import java.sql.Connection;
```
Comment (aboughzela) [inline component/identity/src/test/java/org/exoplatform/services/organization/TestOrganization.java]: Please remove unused import

**Comment by aboughzela** [comment]
LGTM

**Pull request merged**